src/Lykke.Cqrs.MessageCancellation/Lykke.Cqrs.MessageCancellation.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netcoreapp2.1</TargetFramework>
+        <AssemblyName>Lykke.Cqrs.MessageCancellation</AssemblyName>
+        <Version>1.0.1</Version>
+        <Authors>Lykke</Authors>
+        <Product>Lykke.Cqrs.MessageCancellation</Product>
+        <Description>SDK for message cancellation</Description>
+        <PackageIconUrl>https://avatars3.githubusercontent.com/u/14153330?v=3&amp;s=200</PackageIconUrl>
+        <RepositoryUrl>https://github.com/LykkeCity/Lykke.Cqrs.MessageCancellation.git</RepositoryUrl>
+        <RepositoryType>git</RepositoryType>
+        <PackageTags>Lykke Message Messages Cancellation</PackageTags>
+        <Copyright>https://github.com/LykkeCity/Lykke.Cqrs.MessageCancellation/blob/master/LICENSE</Copyright>
+        <PackageProjectUrl>https://github.com/LykkeCity/Lykke.Cqrs.MessageCancellation</PackageProjectUrl>
+        <PackageLicenseUrl>https://github.com/LykkeCity/Lykke.Cqrs.MessageCancellation/blob/master/LICENSE</PackageLicenseUrl>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="JetBrains.Annotations" Version="2018.2.1" />
+        <PackageReference Include="Lykke.Cqrs" Version="8.1.0" />
+        <PackageReference Include="Lykke.Logs" Version="7.2.1" />
+        <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.1.3" />
+    </ItemGroup>
+
```

Comment:
Add SourceLink nuget reference and add pdb to the package - https://github.com/LykkeCity/CommonDotNetLibraries/blob/master/src/Lykke.Common/Lykke.Common.csproj#L48-L56